docs/reference/mpremote.rst

```diff
@@ -42,7 +42,7 @@ The full list of supported commands are:
 
   .. code-block:: bash
 
-      $ mpremote connect <device>
+      $ mpremote connect <device> [command]
```

Comment:
This is documenting the `connect` command, so adding extra `[command]` entries is not correct.